primitives/src/pow.rs

```diff
@@ -36,11 +34,11 @@ impl fmt::LowerHex for CompactTarget {
     #[inline]
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result { fmt::LowerHex::fmt(&self.0, f) }
 }
-impl_to_hex_from_lower_hex!(CompactTarget, |compact_target: &CompactTarget| 8 - compact_target
-    .0
-    .leading_zeros()
-    as usize
-    / 4);
+#[rustfmt::skip]
```

Comment:
```suggestion

```

If I remove this `cargo fmt` does not change the formatting, and changes it back to exactly this if I edit the formatting first.

Same with the other occurrence too.

Reply:
Even with a nightly compiler?

Reply:
Only on nightly.  On stable all the settings in `rustfmt.toml` give warnings like ```Warning: can't set `make_backup = false`, unstable features are only available in nightly channel.```

Reply:
Interesting, I think I've seen this sort of thing before. I'm guessing its because its not a single long line now. Yep confirmed. I'll remove the `skip`, thanks man.